practica2023/src/main/java/com/endava/practica2023/Domain/dto/OrdersDTO.java

```diff
@@ -0,0 +1,27 @@
+package com.endava.practica2023.Domain.dto;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Id;
+import lombok.Data;
+
+import java.time.LocalDateTime;
+
+public record OrdersDTO (
+        Integer eventId,
+        LocalDateTime orderedAt,
+        Integer ticketCategoryId,
+        Integer number_of_tickets,
+        Float order_total_price
```

Comment:
Naming convention in Java is camel case for methods and variables, not how you name them with _ between words